client/src/Components/Info.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Stars from './Stars.jsx';
+import axios from 'axios';
+import styles from './../styles/Info.css';
+
+class Info extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      restaurant : {},
+    }
+  }
+
+  componentDidMount() {
+    const id = window.location.pathname.substring(13);
+    axios.get(`/restaurants/${id}/profile`)
+      .then((response) => {
+        this.setState({
+          restaurant: response.data[0],
+        });
+      });
+  }
+
+  getStars(stars) {
```

Comment:
You're copying this getStars function in two different places. It would be better to save it somewhere (like in its own file) and import it rather than repeating the code.